### PR TITLE
fix some problems in paper and url service

### PR DESCRIPTION
**Pull request by penqee**
url: Modified url format
paper: When it was compatible with the URL, it was found that the parameters of downloadUrl did not match the Android version and was modified.
Removed the download and acquisition of release.apk and beta.apk, they can be obtained directly through port 5000

### CODE REVIEW

File: api/handler/api/paper_service.go [hunk 1]
```diff
@@ -59,19 +64,19 @@ func ListDirFiles(ctx context.Context, c *app.RequestContext) {
...
 	var err error
-	var req api.GetDownloadUrlRequest
-	err = c.BindAndValidate(&req)
-	if err != nil {
-		logger.Errorf("api.GetDownloadUrl: BindAndValidate error %v", err)
-		pack.RespError(c, errno.ParamError.WithError(err))
-		return
-	}
+
+	filepath := c.DefaultQuery("filepath", "/C语言/10份练习.zip")
```
Comment (jiuxia211) [inline api/handler/api/paper_service.go:76]: 为啥不用 c.BindAndValidate(&req) ？

File: api/handler/api/url_service.go [hunk 2]
```diff
@@ -154,22 +155,10 @@ func GetUploadParams(ctx context.Context, c *app.RequestContext) {
 	c.JSON(consts.StatusOK, resp)
 }
 
-// GetDownloadRelease .
-// @router /api/v1/url/release.apk [GET]
-func GetDownloadRelease(ctx context.Context, c *app.RequestContext) {
```
Comment (jiuxia211) [inline api/handler/api/url_service.go:159]: 这里两个 GetDownloadRelease 和 GetDownloadBeta 为什么删除了

Comment (penqee) [reply api/handler/api/url_service.go:159]: 服务器上重定向有点问题，需要在nginx上进行配置，我让/api/v1/url/release.apk的直接到5000端口就好了，如果这个保留着的话还要多配置一些，到时候需要调整也不方便

Comment (jiuxia211) [reply api/handler/api/url_service.go:159]: 明儿开会说一下这个问题

File: api/handler/api/paper_service.go (hunk 1 above)
Comment (penqee) [reply api/handler/api/paper_service.go:76]: 确实，我这里改动不行

**Comment by jiuxia211** [comment]
记得更新接口文档

**Comment by ozline** [comment]
我有一个疑问，为什么你的 commit 记录包含着已经合入仓库的commit？

**Pull request closed without merging**